kits/snapkart/.env.example

```
@@ -0,0 +1,19 @@
+﻿# Lamatic Flow IDs
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The ﻿# Lamatic Flow IDs key has incorrect delimiter

(IncorrectDelimiter)